en/middleware/micrortps.md

```diff
@@ -34,6 +33,25 @@ The *Client* application is also compiled and built into the firmware as part of
 > **Tip** The bridge code can also be [manually generated](micrortps_manual_code_generation.md). Most users will not need to do so, but the linked topic provides a more detailed overview of the build process and can be useful for troubleshooting.
 
 
+## Supported uORB messages
+
+The set of uORB topics that can be supported by the bridge are listed in the **msg/CMakeLists.txt**. You can modify the list to change what uORB messages can be published/subscribed using RTPS in the **.cmake** file (**cmake/configs**) for each target platform.
+
+```cmake
+set(config_rtps_send_topics
+  sensor_combined
+   # Add new topic...
+   )
+
+set(config_rtps_receive_topics
+   vehicle_command
+   # Add new topic...
+   )
+```
+
+> **Note** You can further refine which messages are supported when you call **generate_microRTPS_bridge.py** using `-s` (send) and `-r` (receive) flags. Please see [Manual Generation of the Code](../middleware/micrortps_manual_code_generation.md).
```

Comment:
OK, so based on what you say below this is wrong. We need to convey that the lists are used to specify the set for auto generation and that the arguments are used for manual generation (what it says now is that the arguments "filter down" the list defined in the txt file). 

I would do this as a single sentence at the very top (before what is there now).